vscode/syntaxes/toit.tmLanguage.yaml

```diff
@@ -522,4 +522,24 @@ repository:
           name: comment.line.double-slash.toit
       contentName: comment.line.double-slash.toit
 
+  punctuation:
```

Comment:
Just adding this section doesn't do anything yet.

It would also needed to be added to patterns in line 243.